hawkular-bus-common/src/main/java/org/hawkular/bus/common/consumer/RPCBasicMessageListener.java

```diff
@@ -79,7 +79,7 @@ protected void setMessageSender(MessageProcessor messageSender) {
     }
 
     @Override
-    public void onMessage(Message message) {
+    public void onMessage(javax.jms.Message message) {
```

Comment:
trivial issue - why is this fully qualified (javax.jms.Message) verses just "Message" with the import (the import was also removed). Just curious.